secret_store/src/key_server_cluster/math.rs

```diff
@@ -528,6 +563,65 @@ pub mod tests {
 		result
 	}
 
+	fn run_multiplication_protocol(t: usize, secret_shares1: &[Secret], secret_shares2: &[Secret]) -> Vec<Secret> {
+		let n = secret_shares1.len();
+		assert!(t * 2 + 1 <= n);
+
+		// shares of secrets multiplication = multiplication of secrets shares
+		let mul_shares: Vec<_> = (0..n).map(|i| {
+			let share1 = secret_shares1[i].clone();
+			let share2 = secret_shares2[i].clone();
+			let mut mul_share = share1;
+			mul_share.mul(&share2).unwrap();
+			mul_share
+		}).collect();
+
+		mul_shares
+	}
+
+	fn run_reciprocal_protocol(t: usize, artifacts: &KeyGenerationArtifacts) -> Vec<Secret> {
+		// === Given a secret x mod r which is shared among n players, it is
+		// === required to generate shares of 1/x mod r with out revealing
+		// === any information about x or 1/x.
```

Comment:
nit: can we frame that as a multiplicative inverse instead of `1/x`?

Reply:
Sure :) It is just easier to write `1/x` instead of `multiplicative inverse of x`. Did `inv(x)` sounds good, or it is better to write its 'full name'?

Reply:
`inv(x)` is fine for me